go/vt/vttablet/tabletserver/state_manager_test.go

```diff
@@ -669,6 +669,42 @@ func TestStateManagerNotify(t *testing.T) {
 	sm.StopService()
 }
 
+func TestDemotePrimaryStalled(t *testing.T) {
```

Comment:
could we add tests for when a request is sent to a primary that is stalled demoting, it gets an error in response?

Reply:
Done! Added that part to the test.